contracts/utils/Whitelist.sol

```diff
@@ -0,0 +1,72 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity >=0.6.0 <0.8.0;
+
+import "./AccessControl.sol";
+
+contract Whitelist is AccessControl {
+    using EnumerableSet for EnumerableSet.AddressSet;
+
+    event WhitelistEnabled(address sender);
+    event WhitelistDisabled(address sender);
+    event WhitelistAccountAdded(address account);
+    event WhitelistAccountRemoved(address account);
+
+    bytes32 public constant WHITELIST_ROLE = keccak256("WHITELIST_ROLE");
+
+    bool private _isWhitelistEnabled;
+    EnumerableSet.AddressSet private _whitelist;
+
+    modifier onlyWhitelistOrAdmin() {
+        _onlyWhitelistOrAdmin();
+        _;
+    }
+
+    modifier usingWhitelist() {
+        _usingWhitelist();
+        _;
+    }
+
+    function enableWhitelist() external onlyWhitelistOrAdmin {
+        require(!isWhitelistEnabled(), "Whitelist is already enabled");
+        _isWhitelistEnabled = true;
+        emit WhitelistEnabled(msg.sender);
+    }
+
+    function disableWhitelist() external onlyWhitelistOrAdmin {
+        require(isWhitelistEnabled(), "Whitelist is already disabled");
+        _isWhitelistEnabled = false;
+        emit WhitelistDisabled(msg.sender);
+    }
+
+    function addToWhitelist(address account) external onlyWhitelistOrAdmin {
+        require(!isOnWhitelist(account), "Account is already on the whitelist");
```

Comment:
on -> in. Here and in other places.

Reply:
"is whitelisted" [used](https://github.com/Cerebellum-Network/chainbridge-solidity/pull/4/files#diff-d1d675d04151a0a3ec00bad5e7e2fbea4c4d2e70a75a00f656df8fa08553f5f7R47)